packages/flutter/lib/src/gestures/hit_test.dart

```diff
@@ -54,7 +54,7 @@ class HitTestEntry {
   final HitTestTarget target;
 
   @override
-  String toString() => '$target';
+  String toString() => '$runtimeType($target)';
```

Comment:
You might use `'${describeIdentity(this)}($target)'` instead, since it will add a hash to the runtime type that lets you see if two objects are the same object or not.